subsys/tracing/Kconfig

```diff
@@ -1,225 +1,331 @@
-	default 1024
-	depends on TRACING_ASYNC
-	help
-	  Stack size of tracing thread.
-
-config TRACING_THREAD_WAIT_THRESHOLD
-	int "Tracing thread waiting threshold"
-	default 100
-	depends on TRACING_ASYNC
-	help
-	  Tracing thread waiting period given in milliseconds after
-	  every first packet put to tracing buffer.
-
-config TRACING_BUFFER_SIZE
-	int "Size of tracing buffer"
-	default 2048 if TRACING_ASYNC
-	default TRACING_PACKET_MAX_SIZE if TRACING_SYNC
-	range 32 65536
-	help
-	  Size of tracing buffer. If TRACING_ASYNC is enabled, tracing buffer
-	  is used as a ring buffer to buffer data packet and string packet. If
-	  TRACING_SYNC is enabled, the buffer is used to hold the formated data.
-
-config TRACING_PACKET_MAX_SIZE
-	int "Max size of one tracing packet"
-	default 32
-	help
-	  Max size of one tracing packet.
-
-choice
-	prompt "Tracing Backend"
-	default TRACING_BACKEND_UART
-
-config TRACING_BACKEND_UART
-	bool "Enable UART backend"
-	depends on UART_CONSOLE
-
-	help
-	  Use UART to output tracing data.
-
-config TRACING_BACKEND_USB
-	bool "Enable USB backend"
-	depends on USB
-	depends on TRACING_ASYNC
-	help
-	  Use USB to output tracing data.
-
-config TRACING_BACKEND_POSIX
-	bool "Enable posix architecture (native) backend"
-	depends on TRACING_SYNC
-	depends on ARCH_POSIX
-	help
-	  Use posix architecture to output tracing data to file system.
-
-config TRACING_BACKEND_RAM
-	bool "Enable RAM backend"
-	help
-	  Use a ram buffer to output tracing data which can
-	  be dumped to a file at runtime with a debugger.
-	  See gdb dump binary memory documentation for example.
-
-config RAM_TRACING_BUFFER_SIZE
-	int "Ram Tracing buffer size"
-	default 4096
-	depends on TRACING_BACKEND_RAM
-	help
-	  Size of the RAM trace buffer. Trace will be discarded if the
-	  length is exceeded.
-
-endchoice
-
-config TRACING_BACKEND_UART_NAME
-	string "Device Name of UART Device for UART backend"
-	default "$(dt_chosen_label,$(DT_CHOSEN_Z_CONSOLE))" if HAS_DTS
-	default "UART_0"
-	depends on TRACING_BACKEND_UART
-	help
-	  This option specifies the name of UART device to be used for
-	  tracing backend.
-
-config TRACING_USB_MPS
-	int "USB backend max packet size"
-	default 64
-	depends on TRACING_BACKEND_USB
-	help
-	  USB tracing backend max packet size(endpoint MPS).
-
-config TRACING_HANDLE_HOST_CMD
-	bool "Enable host command handle"
-	select UART_INTERRUPT_DRIVEN if TRACING_BACKEND_UART
-	help
-	  When enabled tracing will handle cmd from host to dynamically
-	  enable and disable tracing to have host capture tracing stream
-	  data conveniently.
-
-config TRACING_CMD_BUFFER_SIZE
-	int "Size of tracing command buffer"
-	default 32
-	range 32 128
-	help
-	  Size of tracing command buffer.
-
-config TRACING_ISR
-	bool "Enable tracing ISRs"
-	default y
-	help
-	  Enable tracing ISRs. This requires the backend to be
-	  very low-latency.
-
-endif
-
-source "subsys/tracing/sysview/Kconfig"
+# Debug configuration options

```

Comment:
looks like you have Windows line termination here..

```
file  subsys/tracing/Kconfig
subsys/tracing/Kconfig: ASCII text, with CRLF line terminators
```

Reply:
Good find, this has now been resolved.